lib/resque/server/views/workers.erb

```diff
@@ -29,7 +29,18 @@
       <td class='process'>
         <% data = worker.processing || {} %>
         <% if data['queue'] %>
+          <% kill_worker = worker.terminal? %>
+          <% unless kill_worker %>
+            <form id="retry-worker-form" method="POST" style="display:none;"><input type="hidden" name="_method" value="POST"></form>
+            <form id="kill-worker-form" method="POST" style="display:none;"><input type="hidden" name="_method" value="DELETE"></form>
+          <% end %>
           <code><%= data['payload']['class'] %></code>
+          <% unless kill_worker %>
+            <a class="kill-tag" href="#" onclick="if(window.confirm('Are you SURE you want to abort this job\'s attempt?')) $('#retry-worker-form').submit()">Abort Attempt</a>
```

Comment:
Isn't retrying something that is added by [resque-retry](https://rubygems.org/gems/resque-retry).  I don't think resque itself has that concept.